### PR TITLE
Added history uuid to seqno stats

### PR DESCRIPTION
**Pull request by martijnvg**

### CODE REVIEW

**Review by jasontedor — APPROVED** [review]
LGTM.

File: core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java [hunk 1]
```diff
@@ -198,7 +198,8 @@ public InternalEngine(EngineConfig engineConfig) {
                         seqNoStats = new SeqNoStats(
                                 SequenceNumbers.NO_OPS_PERFORMED,
                                 SequenceNumbers.NO_OPS_PERFORMED,
-                                SequenceNumbers.UNASSIGNED_SEQ_NO);
+                                SequenceNumbers.UNASSIGNED_SEQ_NO,
+                                getHistoryUUID());
```
Comment (jasontedor) [inline core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java]: I am not comfortable with this invocation when history UUID has not been initialized yet.

**Review by jasontedor — CHANGES_REQUESTED** [review]
Sorry about the premature LGTM, I was trying to leave one comment.

**Comment by martijnvg** [comment]
Closing as this PR is not yet relevant for the current phase of development.

**Pull request closed without merging**